doc/index.rst

```diff
@@ -174,7 +174,6 @@ Strawberry Fields is **free** and **open source**, released under the Apache Lic
    code/sf_apps
    code/sf_ops
    code/sf_utils
-   code/sf_api_client
```

Comment:
Instead, the new `api` directory will have to be included, such that when `make docs` is run, the new directory is documented.

Reply:
I added a `sf_api.rst` as you described, and I _think_ the docs are showing up properly in the expected places - but would appreciate if someone could confirm I did it right!

Reply:
@pauljxtan thanks so much! :blush: Could you move the line with the `api` change here? (such that it is in alphabetical order, this is also the order in which it appears on the website)